src/components/functions/GetUserByID.ts

```diff
@@ -0,0 +1,6 @@
+import usersFromServer from '../../api/users';
```

Comment:
the file name should be in camelCase instead of PascalCase - `getUserById.ts`

PascalCase for files with React components